src/include/cart/types.h

```diff
@@ -71,6 +71,8 @@ typedef struct crt_init_options {
 	 */
 	uint32_t	cio_max_expected_size;
 	uint32_t	cio_max_unexpected_size;
+			/** swim crt index */
+	int		cio_swim_crt_idx;
```

Comment:
this is confusing now. this appears to align with my early comment about having this option, but i dont see this field being used anywhere